python/README.md

```diff
@@ -30,26 +30,27 @@ cd iceberg/python
 pip install -e .
 ```
 
-## Testing
+Development is made easy using [Poetry](https://python-poetry.org/docs/#installation). 
 
-Testing is done using tox. The config can be found in `tox.ini` within the python directory of the iceberg project.
+## Development
```

Comment:
It would be good to have a setup section that instructs you to run `pip install poetry`.

Reply:
I also had to upgrade pip and virtualenv in Ubuntu 20.04 to get poetry working properly.

Reply:
`sudo pip install virtualenv --upgrade` and `sudo python3 -m pip install --upgrade pip`

Reply:
Cool, I've added the commands 👍🏻  btw, some interesting combinations you have there. `sudo` shouldn't be required, and `pip install` and `python3 -m pip install` are equivalent.